src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -411,7 +412,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     completionWork onFailure {
       case e =>
         logger.error(s"Preemption work failed for call ${callKey.tag}! " + e.getMessage, e)
-        scheduleTransition(WorkflowFailed)
+        self ! CheckForWorkflowComplete
```

Comment:
does `CheckForWorkflowComplete` need to be a message?  looks like you could just call the method.